CHANGELOG.md

```diff
@@ -1,9 +1,21 @@
 # Changelog
 
+## 3.0.2-beta
+
+- Custom ClickHouse settings can be set in data source settings. [Allow passing custom ClickHouse settings in datasource](https://github.com/grafana/clickhouse-datasource/pull/366)
```

Comment:
Can we backport this PR to 2.x branch? There is no reason to postpone the release of this improvement